infra/modules/service/draft_documents.tf

```diff
@@ -1,4 +1,5 @@
 resource "aws_s3_bucket" "draft_documents" {
+  count         = startswith(var.service_name, "api-") ? 1 : 0
```

Comment:
I thought about it overnight, and...

I think we should swap this to a `enable_drafts_bucket` parameter on the module

sorry for the run around >_>

Reply:
I'm pretty sure this isn't quite right, but it validates... 
https://github.com/HHS/simpler-grants-gov/pull/3068/commits/09a643a4fe0fdb5923d0ff41cfa732c8bbc0d6ad